man/wacom.man

```diff
@@ -26,9 +28,6 @@ and requires the wacom kernel driver being loaded before this driver starts.
 Please check https://github.com/linuxwacom for latest updates of Wacom X
 and kernel drivers.
 .SH DRIVER-INTERNAL DEVICE HOTPLUGGING
-When input device hotplugging in the X server is enabled and no
-.B InputDevice
-section exists for a compatible tablet device and an
 .B InputClass
```

Comment:
This needs a `The` prefix, otherwise it sounds a bit awkward. (i.e. `The inputclass blah blah`)

Reply:
Understood. I updated the commit for this.